<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chris Engine 1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="OpenGLLayout">
      <property name="spacing">
       <number>3</number>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Position Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="positionStep">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Rotation Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="rotationStep">
        <property name="suffix">
         <string>Â°</string>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>30.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Scale Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="scaleStep">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionGrid"/>
    <addaction name="actionAxis"/>
    <addaction name="actionCorner_Gizmo"/>
    <addaction name="actionWireframe"/>
    <addaction name="actionDraw_Normals"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionBasic_Controls"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add Objects</string>
    </property>
    <addaction name="actionTriangle"/>
    <addaction name="actionQuad"/>
    <addaction name="actionCube"/>
    <addaction name="actionSphere"/>
   </widget>
   <addaction name="menuEditor"/>
   <addaction name="menuAdd"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGrid"/>
   <addaction name="actionAxis"/>
   <addaction name="actionCorner_Gizmo"/>
   <addaction name="actionWireframe"/>
   <addaction name="actionDraw_Normals"/>
   <addaction name="actionRender_MousePick"/>
   <addaction name="actionFrustum_Culling"/>
   <addaction name="actionLOD"/>
   <addaction name="actionSound"/>
   <addaction name="actionPlay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidgetOutliner">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>107</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Scene Outliner</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidgetSceneOutliner">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetDetails">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>500</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Details</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetVerticalDetails">
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEditGOName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="placeholderText">
        <string>none</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetDetails">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>312</width>
          <height>436</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutDetails">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionAxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>World Axis</string>
   </property>
  </action>
  <action name="actionCorner_Gizmo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Corner Gizmo</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
  </action>
  <action name="actionDraw_Normals">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Normals</string>
   </property>
  </action>
  <action name="actionBasic_Controls">
   <property name="text">
    <string>Basic Controls</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Start/Stop the game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRender_MousePick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Render MousePick</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="actionQuad">
   <property name="text">
    <string>Plane</string>
   </property>
  </action>
  <action name="actionCube">
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionFrustum_Culling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Frustum Culling</string>
   </property>
   <property name="toolTip">
    <string>Toggle Frustum Culling</string>
   </property>
  </action>
  <action name="actionLOD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOD</string>
   </property>
   <property name="toolTip">
    <string>Toggle LOD</string>
   </property>
  </action>
  <action name="actionSound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sound</string>
   </property>
   <property name="toolTip">
    <string>Toggles Sound</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
